todo_app/templates/index.html

```diff
@@ -8,11 +8,44 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
+    <div class="col-md-8">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <div class="card">
+        <ul class="list-group list-group-flush">
+          {% for item in items %}
+            <li class="list-group-item d-flex justify-content-between align-items-center">
+              {% if item.status == 'Completed' %}
+                <strike>{{ item.title }}</strike>
```

Comment:
Technically not meant to use `strike` in HTML 5 - it's deprecated.

You want `s` or `del`.

Or if the only difference in the if/else is styling, I personally think it would be neater to do use a conditional class.

```
<span {% if item.status == 'Completed' %} class="text-decoration-line-through"" {% endif %}>
  {{ item.title}}
</span>
```